src/functions/postLogs/application/__tests__/Logger.spec.ts

```diff
@@ -1,19 +1,19 @@
-import { Mock, It, Times } from "typemoq";
-import LogEvent from "../LogEvent";
-import Logger, { LogDelegate } from "../Logger";
+import { Mock, It, Times } from 'typemoq';
+import LogEvent from '../LogEvent';
+import Logger, { LogDelegate } from '../Logger';
 
-describe("Logger", () => {
+describe('Logger', () => {
```

Comment:
.spec.ts files should be ignored, you can add an ignorePatterns: [] array into the eslint.rc containing "**.spec.ts" and it will ignore them